### ISSUE TITLE
TypeError: stringMask.split is not a function

### ISSUE THREAD

**Issue by probil** [probil/v-mask#482]
From @moveax3 originally published at #481 :
> Hey @probil 
> Very thanks for your contribution!
> I did try version 2.2.1, but has the same error:
> 
> > vue.min.js:6 TypeError: stringMask.split is not a function
> >     at stringMaskToRegExpMask (v-mask.js:302)
> >     at updateMask (v-mask.js:401)
> >     at bind (v-mask.js:409)
> >     at fr (vue.min.js:6)
> >     at vue.min.js:6
> >     at Array.sr (vue.min.js:6)
> >     at y (vue.min.js:6)
> >     at f (vue.min.js:6)
> >     at h (vue.min.js:6)
> >     at f (vue.min.js:6)
> 
> I did use the same code as @davismcphee (https://github.com/probil/v-mask/issues/481#issuecomment-633733177).
> I attach to this message archive with an example.
> [tmp.zip](https://github.com/probil/v-mask/files/4681768/tmp.zip)
> 
> 
> Trying in browsers: 
> SafaryVersion 13.1 (15609.1.20.111.8)
> Google Chrome Version 81.0.4044.138 (Official Build) (64-bit)
> Chromium Version 79.0.3917.0 (Developer Build) (64-bit)
> in OsX 10.15.4 (19E287)

**Comment by probil** [comment]
Hey @moveax3
I've just found out what caused the problem. It turned out that your `v-mask.js` differs from what's published on npm.
I suppose you had taken the library from `dist` folder of the repo (which wasn't update due to migration to semantic release).

Next time just use `jsdelivr` instead. I'll remove `dist` folder from the repo so it's not confusing anyone.

Here is working example, I've changed only the path to `v-mask.js`
https://jsbin.com/masezeriso/edit?html,output
